packages/block-library/src/block/edit.js

```diff
@@ -333,6 +354,20 @@ export default function ReusableBlockEdit( {
 					</ToolbarGroup>
 				</BlockControls>
 			) }
+
+			{ hasOverridableBlocks && (
+				<BlockControls>
+					<ToolbarGroup>
+						<ToolbarButton
+							onClick={ resetOverrides }
+							aria-disabled={ overrides ? 'false' : 'true' }
```

Comment:
I remember that `Buttons` are a bit unusual in that you have to pass `disabled` and `__experimentalIsFocusable` in order for it to retain focus.

Pushed a commit that does that.